numpy/f2py/rules.py

```diff
@@ -15,14 +15,14 @@
     get_b_from_python
     if (successful) {
 
-      callfortran
-      if (succesful) {
+      call_fortran
```

Comment:
This one shouldn't be changed I think - ``callfortran`` is used a lot in the rest of this file.